### ISSUE TITLE
FNE90 Production de lien en mode graphe

### ISSUE THREAD

**Issue by benbober** [abes-esr/poc-fne#35]
https://gitlab.com/inventaire/fne-backlog/issues/90

Estimation (nb 1/2 journées) :0

**Comment by gotnc** [comment]
Cela signifie-t-il créer des liens entre entités à partir d'un graphe ?

**Comment by maxlath** [comment]
@gotnc oui, à partir du graphe constitué par les relations entre ces entités, par exemple la relation d'une oeuvre avec son auteur, voir les [échantillons chargés avec l'exploration de la modélisation de relations](https://github.com/abes-esr/poc-fne/issues/112#issuecomment-516788295)

**Comment by jum-s** [comment]
Dans l'exemple ci-dessous lieu est un lien cliquable

![Sélection_009](https://user-images.githubusercontent.com/5363918/68119115-c06a1400-fef9-11e9-8fe7-125c5b911ccd.png)
